db/fixtures/miq_product_features.yml

```diff
@@ -1841,6 +1841,26 @@
         :feature_type: admin
         :hidden: true
         :identifier: event_edit
+  - :name: Event Streams
```

Comment:
@h-kataria Do you have any additional suggestions on the product feature/user role updates for this API?